### PR TITLE
Skip inheritdoc doc blocks.

### PR DESCRIPTION
**Pull request by dereuromark**

### CODE REVIEW

**Review by a-sabaa — APPROVED** [review]

File: Spryker/Sniffs/Commenting/DocBlockConstSniff.php [hunk 1]
```diff
@@ -40,6 +40,10 @@ public function process(File $phpCsFile, $stackPointer)
         $tokens = $phpCsFile->getTokens();
 
         $docBlockEndIndex = $this->findRelatedDocBlock($phpCsFile, $stackPointer);
+        $docBlockStartIndex = $tokens[$docBlockEndIndex]['comment_opener'];
+        if ($this->hasInheritDoc($phpCsFile, $docBlockStartIndex, $docBlockEndIndex)) {
```
Comment (qlmmlp) [inline Spryker/Sniffs/Commenting/DocBlockConstSniff.php:44]: We have some validation of `$docBlockEndIndex` So we might need to move call to `$this->hasInheritDoc()` to a bit lower?

File: Spryker/Sniffs/Commenting/DocBlockConstSniff.php [hunk 2]
```diff
@@ -40,6 +40,10 @@ public function process(File $phpCsFile, $stackPointer)
         $tokens = $phpCsFile->getTokens();
 
         $docBlockEndIndex = $this->findRelatedDocBlock($phpCsFile, $stackPointer);
+        $docBlockStartIndex = $tokens[$docBlockEndIndex]['comment_opener'];
```
Comment (qlmmlp) [inline Spryker/Sniffs/Commenting/DocBlockConstSniff.php:43]: This line duplicates line #69. and might be removed?

**Review by qlmmlp — CHANGES_REQUESTED** [review]
I guess this line can be removed now.

File: Spryker/Sniffs/Commenting/DocBlockConstSniff.php (hunk 1 above)
Comment (dereuromark) [reply Spryker/Sniffs/Commenting/DocBlockConstSniff.php:44]: You are right, good eye - fixed in https://github.com/spryker/code-sniffer/commit/8c4740735c42ab898d4bcdda34e428b2caaad268

**Pull request merged**